_scripts/instruction-widget/templates/install/ubuntu.html

```diff
@@ -12,8 +12,19 @@
       <li>sudo add-apt-repository ppa:certbot/certbot</li>
       <li>sudo apt-get update</li></ol></pre>
     </p>
-</li>
 {{/ppa}}
+{{^ppa}}
+    Enable the universe repository
+    <p>
+    You'll need to make sure Ubuntu universe is in your list of repositories.
+    To do so, run the following commands on the command line on the machine:
+    <pre class="no-before"><ol><li>sudo apt-get update</li>
+      <li>sudo apt-get install software-properties-common</li>
+      <li>sudo add-apt-repository universe</li>
+      <li>sudo apt-get update</li></ol></pre>
+    </p>
+{{/ppa}}
+</li>
```

Comment:
Exxxtreeeemely minor style consistency nit: the `li`s are usually within the `{{}}`, even when they're mutually exclusive. It's because the concept is a flexible list that includes or doesn't include items, vs being conceptually a branching tree with different relevant options at the different steps.

Reply:
Makes sense! I made that change.